data-structures.md

```diff
@@ -221,7 +221,107 @@ Bitfields are a set of bits encoded using a custom run length encoding: rle+.  r
 
 #### `SectorSet`
 
-TODO
+The `SectorSet` is an integer set implemented with a simple array mapped tree.
+Integer indexes range from 0 to infinity (TODO practical bounds / bounds implied by encoding?).
```

Comment:
Is 64 bit int limitation OK?

Reply:
Probably worth setting some maximum based on the maximum number of sectors we expect to impose on miners